src/specs/models/work-request-bodies/WorkRequestMarkerGenes.v1.yaml

```diff
@@ -0,0 +1,32 @@
+title: Get Marker Genes
+description: Work request body for getting the Marker Genes.
+type: object
+properties:
+  name:
+    type: string
+    description: Name of the work. Must be `MarkerHeatmap`.
+    pattern: MarkerHeatmap
+  nGenes:
+    type: number
+    description: Amount of marker genes to get
+  config:
+    type: object
+    properties:
+      resolution:
+        type: number
+        minimum: 0
+        maximum: 2
```

Comment:
this sounds wrong

Reply:
Looks like sensible values. 2 might be a bit excessive but it won't break anything. Besides, it depends on the slider for the resolution values for clustering, so it makes sense to have the same values.